esphome/components/sdp3x/sdp3x.cpp

```diff
@@ -0,0 +1,128 @@
+#include "sdp3x.h"
+#include "esphome/core/log.h"
+#include "esphome/core/helpers.h"
+
+namespace esphome {
+namespace sdp3x {
+
+static const char *const TAG = "sdp3x.sensor";
+static const uint8_t SDP3X_SOFT_RESET[2] = {0x00, 0x06};
+static const uint8_t SDP3X_READ_ID1[2] = {0x36, 0x7C};
+static const uint8_t SDP3X_READ_ID2[2] = {0xE1, 0x02};
+static const uint8_t SDP3X_START_DP_AVG[2] = {0x36, 0x15};
+static const uint8_t SDP3X_STOP_MEAS[2] = {0x3F, 0xF9};
+
+void SDP3XComponent::update() {
+  this->set_timeout("pressure", 5, [this]() { this->read_pressure_(); });
```

Comment:
I don't really see the need for this?

```suggestion
  this->read_pressure_();
```